internal/controllers/rewards.go

```diff
@@ -299,7 +300,8 @@ type UserResponseDevice struct {
 	// Minted is true if the device has been minted on-chain.
 	Minted bool `json:"minted"`
 	// OptedIn is true if the user has agreed to the terms of service.
-	OptedIn bool `json:"optedIn"`
+	OptedIn      bool `json:"optedIn"`
+	VinConfirmed bool `json:"vinConfirmed"`
```

Comment:
```suggestion
	VINConfirmed bool `json:"vinConfirmed"`
```
Go casing, my favorite axe to grind.

This is a good addition. Let's regenerate the Swagger too.